spec/mainadditions.html

```diff
@@ -228,6 +227,15 @@ <h1>Time Zone Identifiers</h1>
         Each available named time zone identifier is associated with exactly one available named time zone.
         Each available named time zone is associated with exactly one primary time zone identifier and zero or more non-primary time zone identifiers.
       </p>
+      <ins class="block">
+      <p>
+        Time zone identifiers are compared using ASCII-case-insensitive comparisons, and are accepted as input in any variation of letter case.
```

Comment:
```suggestion
        Named time zone identifiers are compared using ASCII-case-insensitive comparisons, and are accepted as input in any variation of letter case.
```

Reply:
We don't actually have a definition for "named time zone" or "named time zone identifier". We only have "available named time zone", "offset time zone", and "time zone identifier". In lieu of introducing a new definition, I opted to use the more general "time zone identifiers" here. I didn't want to use "available named time zone identifier" here because even custom time zone IDs should be compared case-insensitively. And offset time zones can't include letters, so "Time zone identifiers are compared using ASCII-case-insensitive comparisons" is correct even though it's less precise than it could be.

Anyway, the alternative is to add a new definition for "named time zone identifier" that includes both built-in IDs ("available named time zone identifiers") and custom IDs ("custom named time zone identifiers"?). But naming the latter will involve some bikeshedding and I wasn't sure it was strictly needed. 

What do you think?  My preference would be to leave this as-is.

Reply:
I find the current text misleading, because comparison logic for offset time zones is distinct from the logic for named time zones. And given a goal of resolving that without introducing any other new terms, I would suggest something like
> A time zone identifier is a _<dfn>normalized time zone identifier</dfn>_ when it is either an available named time zone identifier or can be matched by _|NormalizedUTCOffset|_. Every time zone identifier produced by an implementation must be a normalized time zone identifier, but the corresponding time zone is also identified by input that is an ASCII-case-insensitive match for its normalized identifier, and an offset time zone is additionally identified by input representing the same numeric offset in a form that can be matched by _|TimeZoneUTCOffsetName|_. For example, input of **"america/los_angeles"** and **"America/Los_angeles"** both correspond with the normalized time zone identifier **"America/Los_Angeles"**, and input of **"-08"** and **"−08"** and **"-0800"** all correspond with the normalized time zone identifier **"-08:00"**.

Reply:
I still have serious PTSD from trying to get this text into ECMA-262 in our earlier PR that required 4+ rounds of feedback with the 262 editors. My sense from going through that process was that brevity was more important to them and they were willing to sacrifice details and explanation to keep things shorter. I agree your text above is clearer, but I suspect that it would just be stripped out by the editors later, so I'm loath to add it now as part of this PR.

So if we were to change this, I'd probably want to add a new term instead of adding a lot of new text.

Also, "Time zone identifiers are compared using ASCII-case-insensitive comparisons" is already in the Temporal spec; this PR just moves it to a different paragraph.

Taking these together, I think it's best to merge this text as-is, and then if we want to clarify it, to use a separate editorial PR for that purpose. Sound OK?